GrocifyApp.UI/src/components/EmptyState.jsx

```diff
@@ -0,0 +1,45 @@
+import { ReactSVG } from 'react-svg';
+import PropTypes from 'prop-types';
+
+//Internal components
+
+//Assets & Css
+import BoxIcon from '../assets/box-ic.svg';
+import PlusCircleIcon from '../assets/plus-circle-ic.svg';
+import styles from './EmptyState.module.scss';
+
+//Consts
+import { EmptyStateConsts } from '../consts/ENConsts';
+
+const EmptyState = ({ onCreate, buttonText }) => {
+    return (
+        <div className={styles.emptyStateContainer}>
+            <div className="icon">
+                <ReactSVG className="react-svg icon-color--n600" src={BoxIcon} />
```

Comment:
this icon will always be the same? or do we need to pass the icon?

Reply:
it will always be the box icon